config/permissions.yml

```
@@ -1096,6 +1096,12 @@ feature_layer_groups_alternate_visibility:
         Disable showing layers from different categories at once.
     label: 'Hide other layer categories if overlay from another category is visible'
     loginRequired: true
+feature_map_attribution:
+    description: |-
+        Access copyright of the ProcedureMapSetting see @link \demosplan\DemosPlanCoreBundle\Logic\ApiRequest\ResourceType\DplanResourceType.
```

for all the changes:
I would not link from here to some code.
Maybe something like
```suggestion
        Restrict the access to edit the public map copyright/attribution (in the admin area)
```

---

@salisdemos done here 7cf65d919528f316b9fcb309d1bd66dc57f583b6